bin/install-core

```diff
@@ -120,13 +124,30 @@ mkdir -p "$DOWNLOAD_DIR"
 
 # TODO: uncomment the downloads when done testing other things
 # Download Bitcoin core checksums and signatures in background
-# TODO: these are messy make them legible.
-( cd "$DOWNLOAD_DIR"; #wget --tries=1 --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=SHA256SUMS* --include-directories=bin http://6hasakffvppilxgehrswmffqurlcjjjhd76jgvaqmsg6ul25s7t3rzyd.onion/en/download/ ||
-wget --no-host-directories --cut-dirs=1 --recursive --level=1 --accept=SHA256SUMS* --include-directories=bin https://bitcoincore.org/en/download/; ) & sig_dl=$!
+(
+    cd "$DOWNLOAD_DIR"
+    wget --tries=1 --no-host-directories --cut-dirs=1 --recursive \
+        #--level=1 --accept=SHA256SUMS* --include-directories=bin \
```

Comment:
delete the commented lines now, if bitcoin-core.org has fixed their onion service finally.